gems/pending/disk/modules/MiqLargeFile.rb

```diff
@@ -2,9 +2,7 @@
 require 'util/miq-system'
 
 if Sys::Platform::IMPL == :linux
-  if MiqSystem.arch == :x86
-    require 'large_file_linux'
-  elsif MiqSystem.arch == :x86_64
+  if MiqSystem.arch == :x86_64
```

Comment:
ok, I was wrong.  Clearly, we only require 'large_file_linux' on 32 bit linux, which we never run on.